NastassiaBon/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def data_extraction
+  file = File.new('data.txt')
+  file.read
+end
+
+def mean(town, data)
+  return -1.0 unless data.include? town
```

Comment:
Better use brackets if the method takes arguments - `data.include?(town)`. You can not use brackets for `p`, `puts`, etc., or methods w/o arguments.

Reply:
Also, why -1.0? Why not -1? I need explanation :)